diff --color -Naur pcem-9999-orig/src/CMakeLists.txt pcem-9999/src/CMakeLists.txt
--- pcem-9999-orig/src/CMakeLists.txt	2022-02-16 01:35:43.910889341 -0500
+++ pcem-9999/src/CMakeLists.txt	2022-02-16 01:36:11.428886300 -0500
@@ -84,7 +84,7 @@
         add_library(pcem-plugin-api SHARED ${PCEM_SRC_PLUGINAPI} ${PCEM_PUBLIC_API})
         target_link_libraries(pcem-plugin-api ${PCEM_LIBRARIES})
         target_compile_definitions(pcem-plugin-api PUBLIC ${PCEM_DEFINES})
-        install(TARGETS pcem-plugin-api RUNTIME DESTINATION bin LIBRARY DESTINATION lib ARCHIVE DESTINATION lib)
+        install(TARGETS pcem-plugin-api RUNTIME DESTINATION bin LIBRARY DESTINATION ${INST_PREFIX} ARCHIVE DESTINATION ${INST_PREFIX})
         set(PCEM_LIBRARIES ${PCEM_LIBRARIES} pcem-plugin-api)
 else()
         set(PCEM_EMBEDDED_PLUGIN_API ${PCEM_SRC_PLUGINAPI} ${PCEM_PUBLIC_API})
