diff --color -Naur wmnut-9999-orig/src/wmgeneral.c wmnut-9999/src/wmgeneral.c
--- wmnut-9999-orig/src/wmgeneral.c	2021-11-12 02:03:59.080943157 -0500
+++ wmnut-9999/src/wmgeneral.c	2021-11-12 02:02:01.372955353 -0500
@@ -37,6 +37,7 @@
 
 extern int	Verbose;
 extern char	*upshost;
+Display  *display;
 /*  extern rckeys	*wmnut_keys; */
 char *wname;
 
diff --color -Naur wmnut-9999-orig/src/wmgeneral.h wmnut-9999/src/wmgeneral.h
--- wmnut-9999-orig/src/wmgeneral.h	2021-11-12 02:04:03.499942699 -0500
+++ wmnut-9999/src/wmgeneral.h	2021-11-12 02:01:21.766959457 -0500
@@ -54,7 +54,10 @@
  /* Global variable */
 /*******************/
 
-Display		*display;
+#ifndef WMGENERAL_H_DISPLAY
+#define WMGENERAL_H_DISPLAY
+extern Display		*display;
+#endif
 
   /***********************/
  /* Function Prototypes */
