--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -123,7 +123,6 @@
 	install(TARGETS Previous BUNDLE DESTINATION /Applications)
 else()
 	install(TARGETS Previous RUNTIME DESTINATION ${BINDIR})
-	install(FILES Previous-icon.bmp DESTINATION ${DATADIR})
 	install(FILES ND_step1_v43_eeprom.bin DESTINATION ${BINDIR})
 	install(FILES Rev_1.0_v41.BIN DESTINATION ${BINDIR})
 	install(FILES Rev_2.5_v66.BIN DESTINATION ${BINDIR})
